contracts/MonetarySupervisor.sol

```diff
@@ -140,8 +149,54 @@ contract MonetarySupervisor is Restricted, TokenReceiver { // solhint-disable-li
     }
 
     // helper function for FrontEnd to reduce calls
-    function getParams() external view returns(uint[2]) {
-        return [ltdDifferenceLimit, allowedLtdDifferenceAmount];
+    function getParams() external view returns(uint[3]) {
```

Comment:
LTD related settings should be in a single struct (eg. `LtdParams`), making it simpler to get/set and avoiding the need to prefix the fields.